kernelci/build.py

```diff
@@ -872,3 +900,28 @@ def publish_kernel(kdir, install='_install_', api=None, token=None,
         resp.raise_for_status()
 
     return True
+
+
+def recreate_dir(path):
```

Comment:
Same comment here, this is a trivial function used in only one place inside this module. I don't see any reason why it should be put as a separate function.

Reply:
removed and code included in pull_tarbal()